source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -121,7 +121,8 @@ def test_disruption_on_stop_area_b(self):
         #at first no disruption
         assert 'disruptions' not in stop
 
-        self.send_chaos_disruption_and_sleep("bob_the_disruption", "stopB", "stop_area")
+        self.send_chaos_disruption_and_sleep("bob_the_disruption", "stopB",
```

Comment:
shouldn't you also try journey on date not in the disruption period for the stops?